frontend/src/features/moti/mapSideBar/InventoryTabs.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { Tab, Tabs } from 'react-bootstrap';
+
+interface IInventoryTabsProps {
+  PropertyForm: React.ReactNode;
+}
+
+export enum InventoryTabNames {
+  property = 'property',
+  title = 'title',
+  owner = 'owner',
+  value = 'value',
+  history = 'history',
+  notes = 'notes',
+}
+/**
+ * Tab wrapper, provides styling and nests form components within their corresponding tabs.
+ * @param param0 object containing all react components for the corresponding tabs.
+ */
+const InventoryTabs: React.FunctionComponent<IInventoryTabsProps> = ({ PropertyForm }) => {
+  return (
+    <Tabs defaultActiveKey={InventoryTabNames.property}>
+      <Tab eventKey={InventoryTabNames.property} title="Property">
+        {PropertyForm}
```

Comment:
Is there any immediately value in passing in form components this way?

Reply:
This is one method to avoiding prop drilling.